pkg/boot/kexec/kexec_linux_implemented.go

```diff
@@ -26,8 +34,38 @@ func FileLoad(kernel, ramfs *os.File, cmdline string) error {
 		flags |= unix.KEXEC_FILE_NO_INITRAMFS
 	}
 
+	if isGzip(kernel) {
+		g, err := gzip.NewReader(kernel)
+		if err != nil {
+			return fmt.Errorf("could read kernel gzip header: %v", err)
+		}
+		decompressed, err := ioutil.TempFile("", "kernel")
+		if err != nil {
+			return fmt.Errorf("could not create temp file: %v", err)
```

Comment:
could not create temp file for decompression
?
people might get confused as to what the context is